website/content/docs/secrets/kmip-profiles.mdx

```diff
@@ -0,0 +1,269 @@
+  These profiles define a set of normative constraints for employing KMIP within a particular
+  environment or context of use.
+---
+
+## [Baseline Server][baseline-server]
+1. <details>
+  <summary>Supports the following objects</summary>
+
+    | Object                                                                    | Supported |
+    | ------------------------------------------------------------------------- | :-------: |
+    | Attribute [[KMIP-SPEC] 2.1.1][kmip-spec-2.1.1]                            | ✅        |
+    | Credential [[KMIP-SPEC] 2.1.2)][kmip-spec-2.1.2]                          | ✅        |
+    | Key Block [[KMIP-SPEC] 2.1.3][kmip-spec-2.1.3]                            | ✅        |
+    | Key Value [[KMIP-SPEC] 2.1.4][kmip-spec-2.1.4]                            | ✅        |
+    | Template-Attribute Structure [[KMIP-SPEC] 2.1.8][kmip-spec-2.1.8]         | ✅        |
+    | Extension Information [[KMIP-SPEC] 2.1.9][kmip-spec-2.1.9]                | ✅        |
+    | Profile Information [[KMIP-SPEC] 2.1.19][kmip-spec-2.1.19]                | ✅        |
+    | Validation Information [[KMIP-SPEC] 2.1.20][kmip-spec-2.1.20]             | ✅        |
+    | Capability Information [[KMIP-SPEC] 2.1.21][kmip-spec-2.1.21]             | ✅        |
+
+  </details>
+  2. <details>
+    <summary>Supports the following subsets of attributes</summary>
+
+    | Attribute                                                                | Supported | Notes  |
+    | -------------------------------------------------------------------------| :-------: | :----: |
+    | Unique Identifier [[KMIP-SPEC] 3.1][kmip-spec-3.1]                       | ✅        |        |
+    | Name [[KMIP-SPEC] 3.2][kmip-spec-3.2]                                    | ✅        |        |
+    | Object Type [[KMIP-SPEC] 3.3][kmip-spec-3.3]                             | ✅        |        |
+    | Cryptographic Algorithm [[KMIP-SPEC] 3.4][kmip-spec-3.4]                 | ✅        |        |
+    | Cryptographic Length [[KMIP-SPEC] 3.5][kmip-spec-3.5]                    | ✅        |        |
+    | Cryptographic Parameters [[KMIP-SPEC] 3.6][kmip-spec-3.6]                | ✅        |        |
+    | Digest [[KMIP-SPEC] 3.17][kmip-spec-3.17]                                | ✅        |        |
+    | Cryptographic Usage Mask [[KMIP-SPEC] 3.19][kmip-spec-3.19]              | ✅        |        |
+    | State [[KMIP-SPEC] 3.22][kmip-spec-3.22]                                 | ✅        |        |
+    | Initial Date [[KMIP-SPEC] 3.23][kmip-spec-3.23]                          | ✅        |        |
+    | Process Start Date [[KMIP-SPEC] 3.25][kmip-spec-3.25]                    | ✅        | Supported from vault version 1.11 |
+    | Protect Stop Date [[KMIP-SPEC] 3.26][kmip-spec-3.26]                     | ✅        | Supported from vault version 1.11 |
+    | Activation Date [[KMIP-SPEC] 3.24][kmip-spec-3.24]                       | ✅        |        |
```

Comment:
nit: Activation Date here is out of order based on the kmip-spec number

Reply:
Thank you for your review. All the attributes are listed in the same order as in spec http://docs.oasis-open.org/kmip/profiles/v1.4/os/kmip-profiles-v1.4-os.html#_Toc491431430

Reply:
Huh interesting that they did that.